server/datastore/mysql/policies.go

```diff
@@ -43,8 +43,8 @@ func policyDB(ctx context.Context, q sqlx.QueryerContext, id uint, teamID *uint)
 		fmt.Sprintf(`SELECT
        		p.*,
        		q.name as query_name,
-       		(select count(*) from policy_membership where policy_id=p.id and passes=true) as passing_host_count,
-       		(select count(*) from policy_membership where policy_id=p.id and passes=false) as failing_host_count
+			(select count(*) from policy_membership_history where passes=true and id in (select max(id) as id from policy_membership_history where policy_id=p.id group by host_id, policy_id)) as passing_host_count,
```

Comment:
`group by host_id, policy_id` -> `group by policy_id, host_id` (not sure if it makes any difference, but makes sense to revert it).

Reply:
Could you elaborate on the reasoning behind this?

Reply:
Sorry, meant to say it's a bit more clear to me (because it's by policy here, not by host), but prolly just a nit now that I think about it...